data/korath/korath missions.txt

```diff
@@ -257,3 +257,571 @@ mission "Small Scale Delivery"
+				goto questions
+			label yup
+			action
+				log "Agreed to help a woman named Jenny Jetmaster who couldn't make a delivery. Apparently the delivery is illegal, though it's unlikely to be detected since it's in a region of space not patrolled by any law-abiding governments."
+			`	"Really? Okay, hold on..." looking down, she fidgets with a datapad, perhaps transferring permission for the cargo mission. "Yeah, okay." She looks back up at last. "Transfer is approved." Her crewman starts offloading the fish, preparing to transfer it to your ship.`
+			`	"If the authorities won't let you land, read them the serial number from the cargo container, and they'll open a bay for you. Oh, also, this is an illegal mission, so don't get caught, or it'll cost you. Well, it's unlikely anyone out here will care about laws, but I do believe in full disclosure."`
+			choice
+				`	"I'm not afraid of dodging the cops... where there are none. Give me the cargo, and it'll reach its destination. You have my word."`
+					accept
+				`	"On second thought, I don't think I want this mission. Illegal fish deliveries are too strange for my tastes."`
+					decline
+
+	on enter Atik
+		dialog "It seems you do not have automatic clearance to <planet>. If you talk to the planet, the authorities will probably let you land."
+	on visit
+		dialog "You've arrived at <planet> but you left some of the fish behind. You should take off and wait for your escorts to arrive."
+
+event "know about Korath fish jobs"
+
+
+
+mission "Korath [F02]: A Fishy Proposition"
+	landing
+	name "A fishy proposition"
+	description "Transport <cargo> to <planet> by <date> for <payment>. Avoid getting scanned or landing on high-security planets, or you may be fined."
+	cargo "planetary supplies" 15
+	illegal 50000
+	"apparent payment" 140000
+	deadline 4
+	clearance "The authorities do not respond, but a concealed hangar bay door opens. The meaning is clear: land here."
+	source "La'ab Bamen"
+	destination "New Tortuga"
+	to offer
+		has "Korath [F01]: A Fishy Deal: done"
+	on offer
+		conversation
+			`You approach the asteroid and a crater suddenly opens up, revealing a docking bay below. It's large enough to fit entire human space stations, but there are only a few ordinary ships parked below. When you exit your ship, several threatening-looking men in odd uniforms stare you down. They're armed with bright orange rifles whose scorching heat warps the air around them.`
+			`	A woman in a white dress steps forward, whose face radiates kindness, in sharp contrast to her tactical gear. Atop her silky dress, you see two revolvers and a sizable supply of ammo. There's a knife on each leg, just above her knee-high combat boots, and you think there's another knife behind her neck. A rainbow lock contrasts her otherwise light blond hair, and matches the rainbow feather on a necklace draped around her neck.`
```

Comment:
I need to portray that information, in that paragraph, where Iris is first introduced. Unfortunately, I am no Shakespeare, so this is the best I can come up with for now..

Reply:
This bit of text has been modified several times now. Do people like the latest version?

> You approach the asteroid and a crater suddenly opens up, revealing a docking bay. Entire human space stations could fit in here, but there are only a few ordinary ships parked. When you exit the <ship>, several dire men in uniformed body armor greet you. Each holds a bright orange rifle whose scorching heat warps the air around them. They all stare intensely at you as another figure steps forward.
>      It is a woman who captures your attention, with a warm and open smile on her face. Strapped over the top of her silky white dress you see two revolvers and a sizable supply of ammo. Beyond that there is a knife on each leg, just above her knee-high combat boots, and you think there's another one between her shoulder-blades. A rainbow lock contrasts her otherwise light blond hair, and matches the rainbow feather on a necklace draped around her neck.